phoenix-core/src/main/java/org/apache/phoenix/index/GlobalIndexChecker.java

```diff
@@ -494,8 +476,7 @@ private boolean verifyRowAndRemoveEmptyColumn(List<Cell> cellList) throws IOExce
                     return true;
                 }
             }
-            byte[] rowKey = CellUtil.cloneRow(cell);
-            return verifyRow(rowKey);
+            return false;
```

Comment:
nit: comment to point out that empty column wasn't found so we're considering it unverified.

Reply:
Sure